harbor-cluster/templates/openshift-resource-locker-ns.yaml

```diff
@@ -0,0 +1,57 @@
+ {{- if .Values.openshift.patchNs.enabled }}
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: ResourceLocker
+metadata:
+  name: {{ .Release.Name }}-harbor-cluster-ns-patch
+spec:
+  serviceAccountRef:
+    name: {{ .Release.Name }}-resource-locker
+  patches:
+  - targetObjectRef:
+      apiVersion: v1
+      kind: Namespace
+      name: {{ .Release.Namespace }}
+    patchTemplate: |
+      metadata:
+        annotations:
+          openshift.io/sa.scc.mcs: s0:c29,c4
```

Comment:
what is this annotation for ?

Reply:
It is related to SELinux I think. Openshift generate it for each namespace. I don't think we need to change it. I've delete it.